Added French Press option with a classic recipe

ej-mitchell opened the pull request:
## What

Adding a French Press option with a classic, simple recipe to get people started with using this coffee-making method.

## Notes

* The image of the French Press is upside down when the recipe is displayed. I'm looking into why this is.
* There are some languages I was not comfortable translating. Do we have a preferred way of getting translations?

Commit:
Add French Press recipes and translations

ej-mitchell commented:
<img width="655" alt="Screen Shot 2021-09-19 at 11 24 09 AM" src="https://user-images.githubusercontent.com/26049591/133933145-bd4d1c8f-78dc-4d19-805f-b51366cad1a1.png">
<img width="619" alt="Screen Shot 2021-09-19 at 11 24 15 AM" src="https://user-images.githubusercontent.com/26049591/133933146-b64d6e04-3958-4601-a750-e6259354b22b.png">
<img width="644" alt="Screen Shot 2021-09-19 at 11 24 23 AM" src="https://user-images.githubusercontent.com/26049591/133933147-5ce6fc3f-2ac3-4bef-91a6-7a775347f77c.png">

xpmn commented on src/translation/i18n.json:
```diff
@@ -81,7 +83,8 @@
       "moka": "Kawiarka",
-      "v60": "V60"
+      "v60": "V60",
+      "frenchPress": ""
```
You can just put "FrenchPress" in to translations where you don't know the exact word. I think French Press is an international name

xpmn commented on src/assets/icons/frenchpress.svg:
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Did you get this icon somewhere or draw it by yourself? Because license is important.

Also we need to follow the style of other icons (the line weight is important)

ej-mitchell replied:
I grabbed it off an open source site. Let me see if a license was attached to it.

xpmn replied:
I've added my own drawed frenchpress icon to a master branch, you can remove your file and just pull latest changes

xpmn replied:
https://github.com/2brew/2brew.github.io/blob/master/src/assets/icons/frenchpress.svg

ej-mitchell replied:
Thanks, @rewkha - sorry for the delays in response. I was out hiking and not by my computer. ⛰️

xpmn commented on public/frenchPress.json:
```diff
@@ -0,0 +1,21 @@
+      "water": 235,
+      "grind": 7,
+      "temp": 100,
+      "inverted": true,
```
set it to false, it makes icon in a list of recipes upside down

ej-mitchell replied:
_ah_ okay I was stumped about what was causing that 180-degree rotation.

xpmn commented on package-lock.json:
```diff
@@ -1,8 +1,881 @@
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
Please remove this file from your PR, you didn't add any new package, so this updates comes automatically

xpmn commented on public/build/bundle.js:
```diff
@@ -1 +1,5698 @@
-var app=function(){"use strict";function A(){}const e=A=>A;function t(A){return A()}function n(){return Object.create(null)}function r(A){A.forEach(t)}function s(A){return"function"==typeof A}function i(A,e){return A!=A?e==e:A!==e||A&&"object"==typeof A||"function"==typeof A}function l(e,...t){if(null==e)return A;const n=e.subscribe(...t);return n.unsubscribe?()=>n.unsubscribe():n}function o(A){l ...
+
```
please remove this file from PR, I'll update the bundle by myself

xpmn reviewed (changes requested):
Thank you for your contribution. I left few comments after review. Please look

Commit:
Adding toolversions, del unnecessary file change.

This is to help with folks who use linters or plugins like CoC, which
require a newer version of Node to work properly.

ej-mitchell commented on .tool-versions:
```diff
@@ -0,0 +1 @@
+nodejs 16.0.0
```
I can also add this to a different PR if preferred. I think it's important to keep track of node versions somewhere, especially for those who are utilizing version tracking systems where their plugins (like CoC) rely on them. I chose a very recent version so that said plugins will not throw errors about outdated Node packages, but there might be a different version you'd prefer to use instead.

ej-mitchell commented on src/assets/icons/frenchpress.svg:
```diff
@@ -0,0 +1,18 @@
+<path d="M179.243 74.0383C179.942 72.3603 181.869 71.567 183.547 72.2663C185.225 72.9657 186.018 74.8929 185.319 76.5709L174.706 102.034C174.007 103.712 172.08 104.505 170.402 103.806C168.724 103.106 167.931 101.179 168.63 99.5011L179.243 74.0383Z" fill="black"/>
+<path d="M146.105 56.6246C144.393 55.7274 142.278 56.388 141.381 58.1001C140.483 59.8122 141.144 61.9276 142.856 62.8248L168.62 76.3262C170.332 77.2234 172.447 76.5628 173.344 74.8507C174.242 73.1385 173.581 71.0232 171.869 70.126L146.105 56.6246Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M64.4711 157.585C54.6478 162.528 42.679 158.572 37.7363 148.749C37.3125 147.907 36.9489 147.037 36.6484 146.144L27.0376 117.547C25.5108 113.004 25.6735 108.063 27.4959 103.631C29.3184 99.1992 32.6782 95.5731 36.9588 93.4185L45.4904 89.1258C52.6491 85.5238 60.9456 86.647 66.8337 91.3224L68.238 116.348L64.8041 103.881C64.4197 102.486 63.736 101.192 6 ...
+</svg>
```
I did a rebase, and I think in the process this got co-opted into this PR. I am researching how to fix this, but more advanced Git-fu is unfortunately where I have a weak spot. Do you have any suggestions?

xpmn commented:
Approved and merged. Fixed some small things after all. Thank you very much for your contribution!

ej-mitchell commented:
> Approved and merged. Fixed some small things after all. Thank you very much for your contribution!

thanks so much @rewkha :)
